libs/libc/machine/xtensa/arch_memmove.S

```diff
@@ -396,9 +396,6 @@ memmove:
   _beqz a4, .Lbackdone  # avoid loading anything for zero-length copies
   # copy 16 bytes per iteration for word-aligned dst and unaligned src
   ssa8  a3    # set shift amount from byte offset
-#define SIM_CHECKS_ALIGNMENT  1 /* set to 1 when running on ISS with
```

Comment:
Could you please put the exact warning you are trying to fix into PR description?
Maybe the other option is to
```
#undef SIM_CHECKS_ALIGNMENT
#define SIM_CHECKS_ALIGNMENT 1
```
The comment to define seems to be reasonable

Reply:
@pkarashchenko   I have update the PR description. Please review again

Reply:
Yes, so my question remains. Should we remove the definition or add `undef` before definition.

Reply:
@pkarashchenko I have update the patch, please review again.